packages/asset-swapper/src/utils/alt_mm_implementation_utils.ts

```diff
@@ -211,13 +215,40 @@ export async function returnQuoteFromAltMMAsync<ResponseT>(
         }
     }
 
-    const response = await axiosInstance.post(`${url}/quotes`, data, {
-        headers: { Authorization: `Bearer ${apiKey}` },
-        timeout: maxResponseTimeMs,
-    });
+    // let response: AxiosResponse<any>;
+    const response = await axiosInstance
+        .post(`${url}/quotes`, data, {
+            headers: { Authorization: `Bearer ${apiKey}` },
+            timeout: maxResponseTimeMs,
+        })
+        .catch(err => {
```

Comment:
why not use `try/catch`? if you `await` a response, a failed promise will bubble up as a thrown error

Reply:
is there any benefit to doing so?